apps/web/src/pages/Users/UserInformationPage/components/AddToPoolDialog.tsx

```diff
@@ -130,24 +130,20 @@ const AddToPoolDialog = ({ user, poolCandidates }: AddToPoolDialogProps) => {
     return Promise.reject(result.error);
   };
 
-  const submitForm: SubmitHandler<FormValues> = async (
-    formValues: FormValues,
-  ) => {
+  const submitForm: SubmitHandler<FormValues> = (formValues: FormValues) => {
     const poolsToUpdate = formValues.pools
       .map((poolId) => poolMap.get(poolId))
       .filter(notEmpty);
 
-    const promises = poolsToUpdate.map((pool) => {
-      return requestMutation({
+    const promises = poolsToUpdate.map(async (pool) => {
+      return await requestMutation({
```

Comment:
Doesn't awaiting on the mutation here remove the benefit of using `allSettled` below?  Now they're forced to run serially?

Reply:
That is a good point. Instead, I have removed the async keyword so we don't need to await the code to satisfy the rule.

[remove async for add to pool submission](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11658/commits/45174e98fdad2516513f154743f9cf3549aced70)